generators/delimited/testing/start_test

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# NOTE: this file (and 'delimited') use relative paths. Make sure to 
+# run this script from origins_generators/generators/delimited/.
+
+# Generate new output and compare it with the expected output file. 
+# A diff will not work because the facts may not be generated in 
+# the same order each time. (generator uses dictionaries)
+rm testing/test_output.csv
+./delimited testing/test_input.csv >> testing/test_output.csv
```

Comment:
I think that the expected.csv file should not be sorted ahead of time because this would sort the header. The test_output also serves as a valuable reference for users trying to change the generator, so that they can manually look and see what the output should look like. 

Then again, that might be expecting too little of the user. The user probably knows already what an origins fact file looks like.